crates/sui-graphql-rpc/src/functional_group.rs

```diff
@@ -29,8 +29,9 @@ pub(crate) enum FunctionalGroup {
     /// Transaction and Event subscriptions.
     Subscriptions,
 
-    /// Information about the system that changes from epoch to epoch (protocol config, committee,
-    /// reference gas price).
+    /// Information about the system that changes from epoch to epoch, such as
```

Comment:
this is a bit of a tongue twister, why not just take the doc comment for SystemStateSummary, 

/// Aspects that affect the running of the system that are managed by the validators either
/// directly, or through system transactions.